sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc

```diff
@@ -178,10 +187,13 @@ The number of cycles between when the ready signal is deasserted and when the
 pipe can no longer accept new inputs.
 
 This property is not guaranteed to be respected if the pipe is an inter-kernel
-pipe. The compiler is allowed to optimize the pipe if both sides are visible.
+pipe. 
+
+This property will is only valid when the `protocol` property is *avalon_streaming*
```

Comment:
```suggestion
This property is only valid when the `protocol` property is *avalon_streaming*
```
Also, what does it mean for it to be valid? Will it be ignored otherwise? Is it UB?

Reply:
I think this should be ill-formed, diagnostic required.